setup.py

```diff
@@ -59,6 +59,6 @@ def read(fname):
     install_requires=['sagemaker-containers>=2.2.5', 'retrying==1.3.3'],
     extras_require={
         'test': ['tox', 'flake8', 'pytest', 'pytest-cov', 'pytest-xdist', 'mock', 'sagemaker',
-                 'docker-compose', 'mxnet==1.3.0.post0']
+                 'docker-compose', 'mxnet==1.3.0.post0', 'awscli']
```

Comment:
aws cli is installed in the image, and appears to be a buildspec dependency, not a python test dependency, so it's not clear why you need it here.

Reply:
Sorry that's a typo. I don't need awscli here.

Reply:
looks like awscli still needs to be removed